### ISSUE TITLE
Add support for camera distance

### ISSUE THREAD

**Issue by nadewad** [JetBrains/skija#67]
Considering that Skija is used to power compose on the desktop, there should be some parallel support for existing graphics facilities that are part of the compose API surface. Jetpack compose supports "camera distance" parameters on layer that is useful for handling rotations around the x and y axis (for example, card flip animations). This requires leveraging Sk3DView APIs in skia which would need to be included in the Skija bindings.

https://cs.android.com/androidx/platform/frameworks/support/+/androidx-master-dev:compose/ui/ui/src/desktopMain/kotlin/androidx/compose/ui/platform/SkijaLayer.kt;l=130?q=SkijaLayer

A link to the implementation of camera distance in the Android framework is shown in this link and can be leveraged within the Skija implementation.

**Comment by tonsky** [comment]
Thanks, I’ll put in on the queue

**Comment by MMarco94** [comment]
Any update?

**Comment by tonsky** [comment]
No, sorry. What you are asking seems to be deprecated? https://github.com/google/skia/blob/f65cd9be733039d871e23eea8566bc2827dcbd6f/RELEASE_NOTES.txt#L564-L565
